website/docs/docs/build/data-tests.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Add data tests to your DAG"
 sidebar_label: "Data tests"
-description: "Read this tutorial to learn how to use data tests when building in dbt."
+description: "Configure data tests in dbt to assess the quality of your input data or the resulting datasets."
```

Comment:
sugg using dbt test since this has higher searches per algolia data. data test has no searches

```suggestion
description: "Configure dbt tests to assess the quality of your input data and ensure accuracy in resulting datasets."
```

Reply:
I think we need to start distinguishing between the two. We were very purposeful in renaming this across the docs and starting to educate folks on the difference. How about:
```suggestion
description: "Configure dbt data tests to assess the quality of your input data and ensure accuracy in resulting datasets."
```